docs/specification/1.0.md

```diff
@@ -193,30 +193,29 @@ Similarly, each EHR will decide what FHIR resources to authorize and to prefetch
 
 ### Prefetch Template
 
-A _prefetch template_ is a `read` or `search` request that describes relevant data needed by the CDS Service. To allow for prefetch templates that are dependent upon the particular CDS Service request, prefetch templates may include references to context using _prefetch tokens_. A prefetch token is an identifier in a prefetch template that is replaced by a value from the hook context to construct the FHIR URL used to request the prefetch data. 
+A _prefetch template_ is a `read` or `search` request that describes relevant data needed by the CDS Service. For example, the following is a prefetch template for hemoglobin A1c observations:
 
-The `prefetch` field of a CDS Service description defines the set of prefetch templates for that service, providing a _prefetch key_ for each one that is used to provide the prefetch data in the CDS request.
+```
+Observation?patient={{context.patientId}}&code=4548-4&_count=1&sort:desc=date
+```
 
-Prefetch tokens MUST be delimited by `{{` and `}}`, MUST be named based upon the field they correspond to, and MUST have a primitive value.
+To allow for prefetch templates that are dependent on the workflow context, prefetch templates may include references to context using _prefetch tokens_. In the above example, `{{context.patientId}}` is a prefetch token. A prefetch token is a placeholder in a prefetch template that is replaced by a value from the hook's context to construct the FHIR URL used to request the prefetch data. 
 
-Individual hooks specify which of their `context` fields can be used as prefetch tokens. Only root-level fields with a primitive value within the `context` object are eligible to be used as prefetch tokens.
+Prefetch tokens MUST be delimited by `{{` and `}}`, and MUST contain only the qualified path to a hook context field.
```

Comment:
Add link here to Prefetch tokens section

Reply:
This is already in the prefetch tokens section, is there another section you're thinking of that we should link to here?